### PR TITLE
Grammatical and minor content changes to PEP 546.

### PR DESCRIPTION
**Pull request by Lukasa**
As requested by @haypo.

### CODE REVIEW

**Comment by Lukasa** [comment]
This will likely interest @ncoghlan.

File: pep-0546.txt [hunk 1]
```diff
@@ -12,17 +12,18 @@ Created: 30-May-2017
...
 Rationale
 =========
 
-While Python 2.7 is getting closer to its end-of-line (scheduled for
-2020), it is still used on production and the Python community is still
-responsible for its security. And to facilitate the future adoption of
-:pep:`543`, which will improve security for Python3 users.
+While Python 2.7 is getting closer to its end-of-support date (scheduled for
+2020), it is still used on production systems and the Python community is still
+responsible for its security. This PR will help facilitate the future adoption
```
Comment (alex) [inline pep-0546.txt]: s/PR/PEP/

File: pep-0546.txt [hunk 2]
```diff
@@ -34,37 +35,32 @@ Python Package Index instead is not an acceptable solution.
...
-Python security: give access to the root certificate authorities on
-Windows and macOS by using native APIs, instead of OpenSSL. A side effect
-is that it gives access to certificates installed locally by system
-administrators, allowing to use "company certificates" without having to
-modify each Python application and so validate correctly TLS
-certificates (instead of having to ignore or bypass the TLS certificate
-validation).
+:pep:`543` defines a new TLS API for Python which would enhance Python
+security by giving Python applications access to the native TLS implementations
+on Windows and macOS by using native APIs, instead of OpenSSL. A side effect
```
Comment (alex) [inline pep-0546.txt]: you can just drop "by using native APIs", how else would you access a native TLS implementation :-)

File: pep-0546.txt [hunk 3]
```diff
@@ -34,37 +35,32 @@ Python Package Index instead is not an acceptable solution.
...
-Python 3.0 was released. In practice, many applications still run on
-production on top of Python 2.7. To make the new TLS API more widely
-used, it should be usable on all Python versions currently supported:
-Python 2.7, 3.5, 3.6. Otherwise, some applications would have to wait
-until they drop Python 2 support to be able to use the new TLS API.
-
-Delaying adoption of the PEP 543 API means delaying the adoption for
-security improvements for Python 3 users as well.
+If this backport is not performed, the only baseline implementation that could
+be used would be PyOpenSSL. This is problematic, however, because of the
```
Comment (alex) [inline pep-0546.txt]: no capital P in pyOpenSSL.

File: pep-0546.txt (hunk 1 above)
Comment (Lukasa) [inline pep-0546.txt]: ACK.

File: pep-0546.txt (hunk 2 above)
Comment (Lukasa) [inline pep-0546.txt]: ACK.

File: pep-0546.txt (hunk 3 above)
Comment (Lukasa) [inline pep-0546.txt]: ACK.

**Pull request merged**